processing/generate_figure.py

```diff
@@ -603,28 +604,57 @@ def main():
 
     # Generate figure for T1w and T2w agreement per vendor
     plt.subplots(figsize=(15, 5))
+    # Loop across vendors (create subplot for each vendor)
     for index, vendor in enumerate(list(OrderedDict.fromkeys(vendor_sorted))):
         ax = plt.subplot(1, 3, index + 1)
-        plt.scatter(np.concatenate(CSA_dict[vendor + '_t2'], axis=0),
-                    np.concatenate(CSA_dict[vendor + '_t1'], axis=0),
+        x = np.concatenate(CSA_dict[vendor + '_t2'], axis=0)
+        y = np.concatenate(CSA_dict[vendor + '_t1'], axis=0)
+        plt.scatter(x,
+                    y,
                     s=50,
                     linewidths=2,
                     facecolors='none',
                     edgecolors=vendor_to_color[vendor],
                     label=vendor)
-        plt.xlim(45, 100)
-        plt.ylim(45, 100)
-        plt.gca().set_aspect('equal', adjustable='box')
+        # Define vendor name position
+        legend = ax.legend(loc='lower right', handletextpad=0, fontsize=FONTSIZE)
+        # Change box's frame color to black (to be same as box around linear fit equation)
+        frame = legend.get_frame()
+        frame.set_edgecolor('black')
+        ax.add_artist(legend)
+        # Dynamic scaling of individual subplots based on data
+        offset = 8
```

Comment:
By changing this, it is possible to "zoom" more to data. But I noticed that not for all numbers is the grid "squared".